### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mkovshun**
- [DEMO LINK](https://mkovshun.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://mkovshun.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,71 @@
...
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="Apple" class="nav__item is-active">Apple</a>
+          </li>
```
Comment (TarasHoliuk) [inline src/index.html:31]:
```suggestion
          <li class="nav__item">
            <a href="Apple" class="nav__link is-active">Apple</a>
          </li>
```
it seems like classes are split: `a` should be link and `li` should be item I guess :slightly_smiling_face:

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,82 @@
...
+  width: 100%;
+  height: 100%;
+  flex-shrink: 0;
+  align-items: center;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
```
Comment (TarasHoliuk) [inline src/style.css:38]: default value - no need to set it

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,82 @@
...
+  position: relative;
+}
+
+.nav__item.is-active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  background-color: var(--main-accent-color);
+  height: 4px;
+  width: 36.61px;
```
Comment (TarasHoliuk) [inline src/style.css:79]: 100%

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,82 @@
...
+  height: 100%;
+  transition: color 0.3s;
+  line-height: 60px;
+}
+
+.nav__item:hover {
+  color: var(--main-accent-color);
+}
+
+.nav__item.is-active {
```
Comment (TarasHoliuk) [inline src/style.css:67]:
```suggestion
.is-active {
```
Do not use a combined selector without neccasserity

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,82 @@
...
+  color: var(--main-accent-color);
+  margin: 0;
```
Comment (TarasHoliuk) [inline src/style.css:69]: redundant

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,82 @@
...
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
```
Comment (TarasHoliuk) [inline src/style.css:43]:
```suggestion
.nav__link:not(:first-child) {
```
and remove selector below

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
GJ :+1: 
let's make it even better

File: src/index.html [hunk 7]
```diff
@@ -6,9 +6,71 @@
...
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a href="Apple" class="nav__link is-active">Apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:30]: Need to fix everywhere
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
When I click on the link I got this error
image

**Review by volodymyr-soltys97 — APPROVED** [review]
Well done 🔥

**Pull request closed without merging**